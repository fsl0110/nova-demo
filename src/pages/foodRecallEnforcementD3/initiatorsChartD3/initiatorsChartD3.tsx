import React, { PureComponent, HTMLAttributes } from "react";
import classNames from "classnames";
import { Piechart } from "../../../components";

export interface AppState extends HTMLAttributes<HTMLOrSVGElement> {}

export class InitiatorsChartD3 extends PureComponent<AppState> {
  render() {
    const { className } = this.props;

    const data = [
      { label: "part 1", value: "75" },
      { label: "part 2", value: "50" },
      { label: "part 3", value: "25" }
    ];

    return (
      <div className={classNames(className)}>
        <Piechart
          innerRadius={90}
          outerRadius={100}
          data={data}
          x={400}
          y={400}
        />
      </div>
    );
  }
}

/* export const InitiatorsChartD3Connected = connect(
  (state: AppState, ownProps: OwnProps): StateProps => ({
    isLoading: state.isLoading.INITIATORS,
    hasError: state.hasError.INITIATORS,
    data: state.data.INITIATORS
  }),
  {
    fetchInitiators
  }
)(InitiatorsChartD3);

export default InitiatorsChartD3; */
