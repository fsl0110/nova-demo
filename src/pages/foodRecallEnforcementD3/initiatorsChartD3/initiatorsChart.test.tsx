import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { InitiatorsChartD3 } from "./initiatorsChartD3";

it("renders toMatchSnapshot", () => {
  const mock = shallow(<InitiatorsChartD3 />);
  expect(toJson(mock)).toMatchSnapshot();
});
