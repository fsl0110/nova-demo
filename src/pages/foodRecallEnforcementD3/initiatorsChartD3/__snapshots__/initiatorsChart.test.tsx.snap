// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders toMatchSnapshot 1`] = `
<div
  className=""
>
  InitiatorsChartD3
</div>
`;
